### PR TITLE
sperated supplemental resourcses from required materials

### PR DESCRIPTION
**Pull request by yati1998**
Fixes issue #829

### CODE REVIEW

File: financial-literacy/financial-literacy.md [hunk 1]
```diff
@@ -86,6 +81,12 @@ The most important parts of maintaining a good credit score are:
 2. only allow credit checks when it is absolutely neccessary; too many credit checks lowers your score
 3. maintain a debt to income ratio that is reasonable
 
+### Supplemental Resources
+- https://medasf.org/newsandpress/media-center/
```
Comment (alodahl) [inline financial-literacy/financial-literacy.md]: Please add a quick description of how to use these resources

File: roles-in-tech/developer-advocate.md [hunk 2]
```diff
@@ -62,6 +62,8 @@ Before you jump on the opportunity, there are some very important points to reme
 - There are part time opportunities though — engineers or people learning in the company can help with Devrel to ease into the job earlier.
 - Always be ready to prove your worth — measuring the impact of a developer evangelist is tough, you need to make sure you’re well organised in recording your successes.
 It’s a versatile, morphing and evolving role. 
+
+- Read through [these](https://docs.google.com/presentation/d/1JF2t4VCznUzD2tyW9kuc0Gx8-q0rnpzjDpXxpMFh8DA/edit#slide=id.p) slides.
```
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]: 🎯

File: ethics/ethics.md [hunk 3]
```diff
@@ -30,13 +30,14 @@ Software is advancing in many aspects, from pure technological capabilities to u
 - [ACM Code of Ethics and Professional Conduct](https://ethics.acm.org/)
 - [Introduction to 
 Software Engineering Ethics](https://www.scu.edu/media/ethics-center/technology-ethics/Students.pdf)
```
Comment (alodahl) [inline ethics/ethics.md]: The lesson and guided practice actually do use these materials. Please change it back.

File: financial-literacy/financial-literacy.md [hunk 4]
```diff
@@ -86,6 +81,12 @@ The most important parts of maintaining a good credit score are:
 2. only allow credit checks when it is absolutely neccessary; too many credit checks lowers your score
 3. maintain a debt to income ratio that is reasonable
 
+### Supplemental Resources
+- [Mission Economic Development Agency](https://medasf.org/newsandpress/media-center/)
```
Comment (alodahl) [inline financial-literacy/financial-literacy.md]: Add descriptions to these resources

File: ethics/ethics.md [hunk 5]
```diff
@@ -28,20 +28,20 @@ Software is advancing in many aspects, from pure technological capabilities to u
 ### Materials
 
 - [ACM Code of Ethics and Professional Conduct](https://ethics.acm.org/)
-- [Introduction to 
-Software Engineering Ethics](https://www.scu.edu/media/ethics-center/technology-ethics/Students.pdf)
+- [Introduction to Software Engineering Ethics](https://www.scu.edu/media/ethics-center/technology-ethics/Students.pdf)
```
Comment (alodahl) [inline ethics/ethics.md]: 😄

**Comment by yati1998** [comment]
@alodahl Please do review my latest commit.

File: financial-literacy/financial-literacy.md [hunk 6]
```diff
@@ -84,10 +84,10 @@ The most important parts of maintaining a good credit score are:
 ### Supplemental Resources
 - [Mission Economic Development Agency](https://medasf.org/newsandpress/media-center/)
    - MEDA creates opportunity for habitually underresourced families throughout the San Francisco Bay Area. MEDA is also committed to maintaining the cultural identity and enhancing the resources of the Mission District, as we help every student achieve and every family succeed as the lead agency of the Mission Promise Neighborhood.
-- [Money Essentials](http://money.cnn.com/pf/money-essentials/)
 - [MyMoney](https://www.mymoney.gov/)
+   - This website is a product of the Congressionally chartered Federal Financial Literacy and Education Commission, which is made up of more than 20 Federal entities that are coordinating and collaborating to strengthen financial capability and increase access to financial services for all Americans. 
 - [Ally Wallet Wise](http://www.allywalletwise.com/)
-
+   - Ally Wallet Wise can help you build healthy habits when it comes to your personal finances. Ally Wallet Wise have the tools and information to help you reach your financial goals, whether you’re starting out or starting over.
```
Comment (Kundan28) [inline financial-literacy/financial-literacy.md]: The link and description will look like two different points. So remove `-` that's before the description

Comment (yati1998) [reply financial-literacy/financial-literacy.md]: i have made the description as a subpoint. So shall i remove the subpoint and add it in same as that of link.

Comment (Kundan28) [reply financial-literacy/financial-literacy.md]: Actually it looks like a different point when I saw the preview...

Comment (yati1998) [reply financial-literacy/financial-literacy.md]: But in my file extra - is not present. shall I repush the file?
you are talking about the last one only?

Comment (yati1998) [reply financial-literacy/financial-literacy.md]: @kundan I have checked this file even and it is fine in this file too. It looks like the subpoint of the link.

File: financial-literacy/financial-literacy.md [hunk 7]
```diff
@@ -86,6 +81,13 @@ The most important parts of maintaining a good credit score are:
 2. only allow credit checks when it is absolutely neccessary; too many credit checks lowers your score
 3. maintain a debt to income ratio that is reasonable
 
+### Supplemental Resources
+- [Mission Economic Development Agency](https://medasf.org/newsandpress/media-center/)
+   - MEDA creates opportunity for habitually underresourced families throughout the San Francisco Bay Area. MEDA is also committed to maintaining the cultural identity and enhancing the resources of the Mission District, as we help every student achieve and every family succeed as the lead agency of the Mission Promise Neighborhood.
+- [MyMoney](https://www.mymoney.gov/)
+   - This website is a product of the Congressionally chartered Federal Financial Literacy and Education Commission, which is made up of more than 20 Federal entities that are coordinating and collaborating to strengthen financial capability and increase access to financial services for all Americans. 
+- [Ally Wallet Wise](http://www.allywalletwise.com/)
```
Comment (alodahl) [inline financial-literacy/financial-literacy.md]: 💙

File: roles-in-tech/developer-advocate.md (hunk 2 above)
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]: Move this to the top of ### Lesson.  Right now it looks like a subpoint of "Why take on this job?"

**Comment by alodahl** [comment]
Titles and descriptions are some of the most important parts of a PR... slow down so you don't misspell things.

**Comment by yati1998** [comment]
> Titles and descriptions are some of the most important parts of a PR... slow down so you don't misspell things.

I am really sorry for that. will take care next time.

**Pull request merged**